dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.restHandler.jakarta-1.0.feature

```diff
@@ -0,0 +1,15 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName = io.openliberty.restHandler.jakarta-1.0
```

Comment:
This name looks odd to me because there is no such feature as `jakarta-1.0`. I suggest calling this `io.openliberty.restHandler1.0-servlet5.0`.

Reply:
This feature will get updated to also tolerate servlet-5.0+ when Jakarta 10 releases.  If it needs to be renamed then I suggest renaming to `io.openliberty.restHandler.jakarta`.  The version after the last `-` is intended to version the overall auto-feature for singleton toleration reasons.  But no auto-feature should be a singleton so there is no need to have the `-1.0` on the end of the name.

Reply:
OK, that is a little less confusing.

Reply:
Changed filename to `io.openliberty.restHandler1.0.jakarta`. Thanks!